<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GNSS-Stylus 1.0.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_Base">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Base</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_Messages_Base" columnstretch="0,1,0" columnminimumwidth="0,50,0">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label_LastWarningMessageLabel_Base">
           <property name="text">
            <string>Last warning message:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton_ClearInfoMessage_Base">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_LastInfoMessage_Base">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_LastWarningMessage_Base">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_RTCMMessageCount_Base">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_RTCMMessageCountLabel_Base">
           <property name="text">
            <string>RTCM messages received:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_ClearRTCMCounter_Base">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_LastInfoMessageLabel_Base">
           <property name="text">
            <string>Last info message:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton_ClearWarningMessage_Base">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_LastErrorMessageLabel_Base">
           <property name="text">
            <string>Last error message:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_LastErrorMessage_Base">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_ClearErrorMessage_Base">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_Base">
         <item>
          <widget class="QPushButton" name="pushButton_ShowMessageWindow_Base">
           <property name="text">
            <string>Show message window</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_SuspendThread_Base">
           <property name="text">
            <string>Suspended</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_Thread_Base">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="label__Comport_Base">
           <property name="text">
            <string>Com port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ComPort_Base">
           <property name="text">
            <string>\\.\COM</string>
           </property>
           <property name="maxLength">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_StartThread_Base">
           <property name="text">
            <string>Start thread</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_TerminateThread_Base">
           <property name="text">
            <string>Terminate thread</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_RoverA">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Rover A</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_Messages_RoverA" columnstretch="0,1,0" columnminimumwidth="0,50,0">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label_LastWarningMessageLabel_RoverA">
           <property name="text">
            <string>Last warning message:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton_ClearInfoMessage_RoverA">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_LastInfoMessage_RoverA">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_LastWarningMessage_RoverA">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_RELPOSNEDMessageCount_RoverA">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_RTCMMessageCountLabel_RoverA">
           <property name="text">
            <string>RELPOSNED-messages received:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_ClearRTCMCounter_RoverA">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_LastInfoMessageLabel_RoverA">
           <property name="text">
            <string>Last info message:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton_ClearWarningMessage_RoverA">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_LastErrorMessageLabel_RoverA">
           <property name="text">
            <string>Last error message:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_LastErrorMessage_RoverA">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_ClearErrorMessage_RoverA">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_Middle_RoverA">
         <item>
          <widget class="QPushButton" name="pushButton_ShowMessageWindow_RoverA">
           <property name="text">
            <string>Show message window</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ShowRELPOSNEDForm_RoverA">
           <property name="text">
            <string>Show RELPOSNED-window</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_SuspendThread_RoverA">
           <property name="text">
            <string>Suspended</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_Thread_RoverA">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="label__Comport_RoverA">
           <property name="text">
            <string>Com port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ComPort_RoverA">
           <property name="text">
            <string>\\.\COM</string>
           </property>
           <property name="maxLength">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_StartThread_RoverA">
           <property name="text">
            <string>Start thread</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_TerminateThread_RoverA">
           <property name="text">
            <string>Terminate thread</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_RoverB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Rover B</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_Messages_RoverB" columnstretch="0,1,0" columnminimumwidth="0,50,0">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label_LastWarningMessageLabel_RoverB">
           <property name="text">
            <string>Last warning message:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton_ClearInfoMessage_RoverB">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_LastInfoMessage_RoverB">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_LastWarningMessage_RoverB">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_RELPOSNEDMessageCount_RoverB">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_RTCMMessageCountLabel_RoverB">
           <property name="text">
            <string>RELPOSNED-messages received:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_ClearRTCMCounter_RoverB">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_LastInfoMessageLabel_RoverB">
           <property name="text">
            <string>Last info message:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton_ClearWarningMessage_RoverB">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_LastErrorMessageLabel_RoverB">
           <property name="text">
            <string>Last error message:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_LastErrorMessage_RoverB">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_ClearErrorMessage_RoverB">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_Middle_RoverB">
         <item>
          <widget class="QPushButton" name="pushButton_ShowMessageWindow_RoverB">
           <property name="text">
            <string>Show message window</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ShowRELPOSNEDForm_RoverB">
           <property name="text">
            <string>Show RELPOSNED-window</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_SuspendThread_RoverB">
           <property name="text">
            <string>Suspended</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_Thread_RoverB">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="label__Comport_RoverB">
           <property name="text">
            <string>Com port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ComPort_RoverB">
           <property name="text">
            <string>\\.\COM</string>
           </property>
           <property name="maxLength">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_StartThread_RoverB">
           <property name="text">
            <string>Start thread</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_TerminateThread_RoverB">
           <property name="text">
            <string>Terminate thread</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_ShowEssentialsWindow">
        <property name="text">
         <string>Show &quot;Essentials&quot;-window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ShowPostProcessingWindow">
        <property name="text">
         <string>Show post-processing window</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
